engine/config.go

```diff
@@ -90,6 +90,11 @@ type DatasetDetail struct {
 	Value string `yaml:"value"`
 }
 
+type ResourcesServer struct {
+	URL       YAMLURL `yaml:"url" validate:"url"`
+	Directory string  `yaml:"directory"`
```

Comment:
hier kan denk ik `validate:"dir"` of `validate:"dirpath"` bij (even testen)

Reply:
Done, ook `validate:"required_without_all"` toegevoegd zodat altijd een van de twee aanwezig moet zijn.